include/roaring/roaring64.h

```diff
@@ -92,6 +93,14 @@ roaring64_bitmap_t *roaring64_bitmap_of_ptr(size_t n_args,
         &((const uint64_t[]){0, __VA_ARGS__})[1])
 #endif
 
+/**
+ * Create a new bitmap by stealing containers from a 32 bit roaring bitmap.
+ *
+ * After calling this function, the original bitmap will be empty, and the
+ * returned bitmap will contain all the values from the original bitmap.
+ */
+roaring64_bitmap_t *roaring64_bitmap_steal_roaring32(roaring_bitmap_t *r);
```

Comment:
Naming suggestion: `roaring64_bitmap_move_from_roaring32`. I posit that `move` is more commonly used than `steal`.